src/hb-ot-meta-table.hh

```diff
@@ -62,6 +69,34 @@ struct meta
 {
   static constexpr hb_tag_t tableTag = HB_OT_TAG_meta;
 
+  struct accelerator_t
+  {
+    void init (hb_face_t *face)
+    { table = hb_sanitize_context_t ().reference_table<meta> (face); }
+    void fini () { table.destroy (); }
+
+    hb_blob_t *reference_entry (hb_tag_t tag) const
+    { return table->dataMaps.lsearch (tag, Null (DataMap)).reference_entry (table.get_blob ()); }
+
+    unsigned int get_entries (unsigned int      start_offset,
+			      unsigned int     *count,
+			      hb_ot_metadata_t *entries) const
+    {
+      unsigned int entries_count = table->dataMaps.len;
+      if (count && *count)
+      {
+	unsigned int len = hb_min (entries_count - start_offset, *count);
```

Comment:
Huh. What if entries_count is smaller than start_offset?

I'm fairly sure we do these a bit nicer in other plcaes.  If not, we *should*.  Basically, create an hb_array_t for the whole array first.  Then make a sub_array of it with start_offset and *count, then iterate over that and copy, maybe just using hb_copy() if feasible.  Yes, I think we should change all our uses to that...

Reply:
Done I guess